package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "kimarie-bonuses",
+  "version": "1.0.0",
+  "description": "Bundle bonuses module (primary)",
+  "main": "index.js",
+  "scripts": {
+    "seed": "node database/scripts/seed.js",
+    "start": "nodemon server/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/KimarieHB/kimarie-bonuses.git"
+  },
+  "author": "KimarieHB",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/KimarieHB/kimarie-bonuses/issues"
+  },
+  "homepage": "https://github.com/KimarieHB/kimarie-bonuses#readme",
+  "dependencies": {
```

Comment:
Same comment as before re listing nodemon here. Whatever you end up doing in your bonuses repo should be mirrored here or vice versa.

Reply:
Added it to `package.json` `"devdependencies"`